rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -304,6 +307,62 @@ def export_c2_manifest_to_local_file(self, set_id):
         _logger.info(f'Manifest Exported to local file:')
         _logger.warning(f'  {local_path}')
 
+    def generate_local_c1_manifest(self):
+        """
+        Creates a new C1 Manifest locally
+        :return:
+        """
+
+        last_run_time = self.dao.get_last_successful_runtime(GenomicJob.C1_PARTICIPANT_WORKFLOW)
+
+        if last_run_time is None:
+            last_run_time = datetime.datetime(2020, 7, 27, 0, 0, 0, 0)
```

Comment:
Since last_run_time will be used to find/filter for new C1 participants (line 324), just want to confirm we really only care about the M/D/Y here and there are no timezone offset issues (e.g, UTC vs. "midnight Eastern" regarding when the PTSC release happens), like we had with the cutoff datetime for cohort assignments.

Reply:
In actuality, we want to pick up any participant that has answered the reconsent-review question yes. This default time is more to limit the query than to actually filter out any participants before this time. Once the first run happens, `last_run_time` is set with the UTC time from the `start_time` of that first run. This will pick up the participants that have answered the reconsent-review question since this `start_time`.